scene/2d/tile_map.cpp

```diff
@@ -1828,7 +1820,6 @@ void TileMap::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::TRANSFORM2D, "cell_custom_transform"), "set_custom_transform", "get_custom_transform");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "cell_half_offset", PROPERTY_HINT_ENUM, "Offset X,Offset Y,Disabled,Offset Negative X,Offset Negative Y"), "set_half_offset", "get_half_offset");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "cell_tile_origin", PROPERTY_HINT_ENUM, "Top Left,Center,Bottom Left"), "set_tile_origin", "get_tile_origin");
-	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "cell_y_sort"), "set_y_sort_enabled", "is_y_sort_enabled");
```

Comment:
Note: If we're choosing to preserve compatibility, this should likely be kept as a deprecated property too.. But I'm starting to question whether asking to keep YSort as deprecated is a good idea. This is for 4.0 and we've broken the API is way more significant ways, not sure that it's worth it to keep stubs where it should be fairly easy for users (if documented) to review their scenes for uses a "YSort" named nodes that changed to `Node2D` type, and tick a property.

Reply:
Considering 4.0 is the time to break the API, that probably Godot users will have to do some work to port their projects, even with some tools to help them (new GDScript, many changes in resources, and more to come), that'd be a good time to remove YSort, considering the feature completely supersedes it.

As a game dev, I'd personally prefer an article in the docs detailing replaced features and steps to update your project than a deprecation warning. Also, going from Godot 3 to 4 tells me my project aren't compatible with the new version anyway so I wouldn't even try to port e.g. a commercial project.

Reply:
Reduz said in IRC that we should still deprecate YSort since we still want to be able to open Godot 3 scenes in Godot 4. As such I've properly implemented the deprecated YSort this time. Let me know if there are any changes!